frontend/public/components/container.jsx

```diff
@@ -108,6 +108,20 @@ const Env = ({env}) => {
   </table>;
 };
 
+// Split image string into the Docker image string and tag.
+const getContainerImageAndTag = image => {
```

Comment:
nit: I would call this something like `splitImageString` or `getImageNameAndTag`